### PR TITLE
Milestone 1: Getting started

### PR DESCRIPTION
**Pull request by SamroodAli**
I created a react app using `create-react-app` to start the calculator app
- [x]  Added dependencies `Big.js` and `propTypes`
- [x] Added github actions and linters
- [x] Deployed to heroku and is available [here](https://samrood-calculator.herokuapp.com/)
Please review my code.
Thanks for your time and consideration.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please, make sure that linter checks are not failing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import 'app.css';
```
Comment (rindrajosia) [inline src/index.js:1]: Please check this line. it should be ` import './app.css' `. That the reason why you cannot deploy your APP.

**Comment by SamroodAli** [comment]
Thank you @rindrajosia for reviewing my code. The import css file was causing a problem. Thanks for solving that.
I am requesting another code review.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,87 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
 
-### Deployment
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+This project is [MIT](lic.url) licensed.
```
Comment (rindrajosia) [inline README.md:85]: - [ ] [REQUIRED] This link is broken, please fix it

File: README.md [hunk 3]
```diff
@@ -1,70 +1,87 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+- A browser
+- Node js  ([install node](https://nodejs.org/en/download/))
+
+### Setup
+1. Clone repository: 
+```sh
+  $ git clone https://github.com/SamroodAli/books-library.git
```
Comment (rindrajosia) [inline README.md:21]: - [ ] [REQUIRED] I think, this should be: $ git clone https://github.com/SamroodAli/react-calculator.git

File: README.md [hunk 4]
```diff
@@ -1,70 +1,87 @@
...
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samrood-ali/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 🤝 Contributing
 
-### Analyzing the Bundle Size
+Contributions, issues, and feature requests are welcome!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Feel free to check the [issues page](issues/).
```
Comment (rindrajosia) [inline README.md:71]: - [ ] [REQUIRED] The issue link is broken. Please fix it.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [ ] Unnecessary files or assets, such as the React svg Logo, should be deleted.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/)). Please check the public folder because logo512.png is still present there.

- [ ] Please make sure that provide correct information in your readme file. Keep in mind that it is the first thing users check.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by oloomoses — APPROVED** [review]
Hi @SamroodAli, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamroodAli** [comment]
Thanks @oloomoses  for reviewing my code.

**Pull request merged**